api/pkg/s3/datastore/ceph/ceph.go

```diff
@@ -0,0 +1,305 @@
+package ceph
+
+import (
+	"bytes"
+	"context"
+	"crypto/md5"
+	"encoding/base64"
+	"fmt"
+
+	"io"
+	"io/ioutil"
+	"time"
+
+	"github.com/Click2Cloud/s3client"
+	. "github.com/Click2Cloud/s3client"
+	"github.com/Click2Cloud/s3client/models"
```

Comment:
Thanks for the PR.  Any reason why these can’t be added to OpenSDS repo directly?

Reply:
Do you mean adding this s3client under opensds org?

Reply:
Yes

Reply:
@leonwanghui @xing-yang shall i go ahead with changing the repo to opensds in vendor and ceph.go?

Reply:
@leonwanghui @xing-yang shall i go ahead with changing the repo to opensds in vendor and ceph.go?

Reply:
Please use https://github.com/webrtcn/s3client directly and only add additional changes to OpenSDS.